managing-cf/audit-events.html.md.erb

```diff
@@ -0,0 +1,214 @@
+---
+title: Audit Events
+owner: CAPI
+---
+
+<strong><%= modified_date %></strong>
+
+## Overview
+Audit events help Cloud Foundry operators monitor actions taken against resources (such as applications) via
+user or system actions.
+
+### Differences from other types of events
+Despite being similar in name, audit events are different from [Cloud Foundry Usage Events](./usage-events.html).
+Unlike usage events, audit events are **not** suitable for application and service billing.
+
+Another tool to help administrators monitor user actions on Cloud Foundry is [Security Event Logging](../../loggregator/cc-uaa-logging.html).
+Unlike Security Event Logs, audit events do **not** cover all API endpoints, but provide more details about a user's actions.
+
+
+### When are events created?
+
+Most events are created when a user acts upon a resource.
+For example, a user issuing a `cf stop APP_NAME` command will cause an `audit.app.stop` event to be created and associated with that user.
+
+Service Events (`audit.service*`) are created after the service broker has approved an interaction.
+
+Crash events (`audit.app.crash`) are created when Diego marks an application instance as crashed.
+
+Blob events (`blob.remove_orphan`) are created by a daily clean up job and do not represent user action.
+
+### Considerations
+Audit events can be created at any point during the execution of the action they describe. This means the action associated with the event is **not** guaranteed
+to have succeeded.
+
+DB migrations may result in the truncation of the audit events table, as the table tends to get quite large.  This means that a deployment with such a migration
```

Comment:
reword this to make it clear that this is not common and it will be detailed in release notes

Reply:
https://github.com/cloudfoundry/docs-running-cf/pull/60/commits/04694449662d6e0cbf66445e09770204cb416424#diff-ba05cd371854e68a073991abb3da83e4R206